miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -411,6 +407,40 @@ open class MiniAppMessageBridge {
         bridgeExecutor.postError(callbackId, "${ErrorBridgeMessage.ERR_SHARE_CONTENT} ${e.message}")
     }
 
+    @VisibleForTesting
+    internal fun onSendJsonToHostApp(callbackObj: CallbackObj)  {
+        sendJsonToHostApp(
+            content = callbackObj.param,
+            onSuccess = { value ->
+                bridgeExecutor.postValue(callbackObj.id, value.toString())
+            },
+            onError = { message ->
+                bridgeExecutor.postError(
+                    callbackObj.id,
+                    message
+                )
+            }
+        )
+    }
```

Comment:
I can see a couple of things wrong here, we are not suppose return the param here, we should parse the contents in the param and send it
```
private fun onSendJsonToHostApp(callbackId: String, jsonStr: String) = try {
        val callbackObj = Gson().fromJson(jsonStr, JsonInfoCallbackObj::class.java)
        val successCallback = { info: String ->
            bridgeExecutor.postValue(callbackId, info)
        }
        val errorCallback = { message: String ->
            bridgeExecutor.postError(
                callbackId, "${ErrorBridgeMessage.ERR_UNIVERSAL_BRIDGE} $message"
            )
        }

        sendJsonToHostApp(callbackObj.param.jsonInfo.content, successCallback, errorCallback)
    } catch (e: Exception) {
        bridgeExecutor.postError(callbackId, "${ErrorBridgeMessage.ERR_UNIVERSAL_BRIDGE} ${e.message}")
    }
``` 

The object is suppose to look like this
```
@Keep
internal data class JsonInfo(val content: String)

@Keep
internal data class JsonInfoCallbackObj(val param: JsonInfoParam) {

    @Keep
    internal data class JsonInfoParam(val jsonInfo: JsonInfo)
}
```